tests/test_utils/test_path_helpers.py

```diff
@@ -316,3 +314,33 @@ def test_update_config_with_paths(
         updated_config["export_paths"]
         == {"export_folder": "R:/DAP_emulation/outgoing_export/"},
     )
+
+def test_validate_config_strings_success(config):
+    """Test validate_config_strings function with valid survey type and platform."""
+    config["survey"]["survey_type"] = "PNP"
+    config["global"]["platform"] = "network"
+    updated_config = validate_config_strings(config)
+    assert updated_config["survey"]["survey_type"] == "PNP", "Survey type should be PNP"
+    assert updated_config["global"]["platform"] == "network", "Platform should be network"
+
+    config["survey"]["survey_type"] = "PNP"
+    config["global"]["platform"] = "s3"
+    updated_config = validate_config_strings(config)
+    assert updated_config["survey"]["survey_type"] == "PNP", "Survey type should be PNP"
+    assert updated_config["global"]["platform"] == "s3", "Platform should be s3"
+
+
+def test_validate_config_strings_invalid_survey_type(config):
+    """Test validate_config_strings function with invalid survey type."""
+    config["survey"]["survey_type"] = "invalid_type"
+    config["global"]["platform"] = "network"
+    with pytest.raises(ValueError, match="Survey type INVALID_TYPE is not valid- it must be one of \['BERD', 'PNP'\]"):
```

Comment:
This might be me being fussy, with the error that is raised when `config["global"]["platform"]` is invalid, can we update the error message so that it reads easier for end users, for exmple like this:

"Survey type INVALID_TYPE is not valid- it must be either BERD or PNP"

My reasoning is, if I was an end user with limited/no Python experience, I might be confused seeing the current message.